engine/src/main/java/org/terasology/rendering/dag/stateChanges/SetDepthTest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.dag.stateChanges;
+
+import static org.lwjgl.opengl.GL11.GL_DEPTH_TEST;
+import org.terasology.rendering.dag.RenderPipelineTask;
+import org.terasology.rendering.dag.StateChange;
+import org.terasology.rendering.dag.tasks.DisableCapabilityTask;
+import org.terasology.rendering.dag.tasks.EnableCapabilityTask;
+
+/**
+ * TODO: Add javadocs
+ */
+public final class SetDepthTest extends SetCapability {
+    private static final int CAPABILITY = GL_DEPTH_TEST;
+    private static StateChange defaultInstance = new SetDepthTest(false);
+    private static RenderPipelineTask enablingTask;
+    private static RenderPipelineTask disablingTask;
+
+    public SetDepthTest(boolean enabled) {
+        super(enabled);
+    }
+
+    @Override
+    public StateChange getDefaultInstance() {
+        return defaultInstance;
+    }
+
+    @Override
+    protected RenderPipelineTask getDisablingTask() {
```

Comment:
In line with comments in the previous commit, this and getEnablingTask() should be removed.